easybuild/easyblocks/g/gcc.py

```diff
@@ -203,6 +203,13 @@ def __init__(self, *args, **kwargs):
             self.log.warning('Setting withnvptx to False, since we are building on a RISC-V system')
             self.cfg['withnvptx'] = False
 
+        # define list of subdirectories in search paths of module load environment
+        self.module_load_environment.PATH = ['bin']
+        self.module_load_environment.LD_LIBRARY_PATH = ['lib', 'lib64']
+        # GCC can find its own headers and libraries but the .so's need to be in LD_LIBRARY_PATH
+        self.module_load_environment.CPATH = []
```

Comment:
Rather than setting `CPATH` explicitly, this should be changed to using the utility functions being added in:
* https://github.com/easybuilders/easybuild-framework/pull/4655

Reply:
fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/3584/files/04dc2b383f7116d218e7fe984ae47cd7ab063eed..9926576e2256d421d0a0cc768c2d6ca85d6b7119